src/tools/rbd/ArgumentTypes.cc

```diff
@@ -293,6 +293,12 @@ void add_size_option(boost::program_options::options_description *opt) {
      "image size (in M/G/T)");
 }
 
+void add_sparse_size_option(boost::program_options::options_description *opt) {
+  opt->add_options()
+    (IMAGE_SPARSE_SIZE.c_str(), po::value<ImageSparseSize>(),
```

Comment:
I would just vote to re-use ```ImageObjectSize``` here so that you don't need to create another validator that does the same thing.